tests/commands/test_pack.py

```diff
@@ -71,7 +78,10 @@ def bundle_yaml(tmp_path):
     bundle_path.write_text("{}")
     content = {}
 
-    def func(*, name):
+    def func(*, name, base_content: Optional[Dict[str, Any]] = None):
+        if base_content:
+            for key, value in base_content.items():
```

Comment:
minor but can you do `content.update(base_content)` here?

Reply:
That's the very thing I was looking for, but last night it just *was not* coming to mind. Thank you!